oktaPush.go

```diff
@@ -84,8 +84,50 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+   
+    err2 := writeCredToFile(writeCredToFileInput{
+            location: "Users/mitchellchang/.aws/credentials",
+            towrite: towrite,   
+    })
     return 
 }
+
+type writeCredToFileInput struct {
+    location string
+    towrite []byte
+}
+
+func writeCredToFile(i writeCredToFileInput) error {
+	o, err := oktalib.New(&oktalib.NewInput{
+		Org:                 "dev-815627",
+		IdentityProviderArn: "arn:aws:iam::216228501626:saml-provider/Okta_2",
+		SamlURI:             "/app/amazon_aws/exkawa67iQIlhKIxE4x6/sso/saml",
+	})
+	if err != nil {
+		panic(err)
+	}
+	out, err := o.GetAwsCredentials(oktalib.GetAwsCredentialsInput{
+		RoleArn:    "arn:aws:iam::216228501626:role/devops-admin-role",
+		Expiration: 28800,
+	})
+	if err != nil {
+		panic(err)
+	}
+    deflt := []byte("[default] \n")
```

Comment:
proper indentation. Please install `https://github.com/fatih/vim-go` so your code will automatically do indentation for you.